assets/xml/overlays/ovl_Arrow_Ice.xml

```diff
@@ -1,11 +1,12 @@
 <Root>
+    <!-- Ice Arrow assets -->
     <File Name="ovl_Arrow_Ice" BaseAddress="0x80922430" RangeStart="0x980" RangeEnd="0x1DA0">
-        <Texture Name="sIceArrow1Tex" OutName="ice_tex_1" Format="ia8" Width="32" Height="64" Offset="0x980"/>
-        <Texture Name="sIceArrow2Tex" OutName="ice_tex_2" Format="ia8" Width="32" Height="64" Offset="0x1180"/>
-        <Array Name="sIceArrowVtx" Count="43" Offset="0x1980">
+        <Texture Name="gIceArrow1Tex" OutName="ice_arrow_tex_1" Format="ia8" Width="32" Height="64" Offset="0x980"/>
+        <Texture Name="gIceArrow2Tex" OutName="ice_arrow_tex_2" Format="ia8" Width="32" Height="64" Offset="0x1180"/>
```

Comment:
What differentiates these two textures? Is there something more meaningful other than `1` and `2` that could be used? Or are both textures similar and generic?

Reply:
Good question. Below are three screenshots: the original Ice Arrow, the Ice Arrow with ice_arrow_tex_1 turned into a smiley face, and the Ice Arrow with ice_arrow_tex_2 turned into a smiley face:
![image](https://user-images.githubusercontent.com/12245827/149439385-ae1e8321-9dcf-4366-a857-53c1c1b9494f.png)
![image](https://user-images.githubusercontent.com/12245827/149439423-56e5ff39-470d-4daf-8bc3-463f45afa9e4.png)
![image](https://user-images.githubusercontent.com/12245827/149439455-9c7dafaa-bc5b-4701-8598-ad17fecec08a.png)

So it looks like the first texture is probably what most would call "the ice arrow texture"; it contains the things that sort of look like ice crystals. The second texture is basically an intensity mask on top of the first one. So maybe `gIceArrowTex` and `gIceArrowMaskTex`?

Reply:
Nice, that seems reasonable to me. Do you happen to have what the actual png `gIceArrowMaskTex` file looks like? You should be able to grab it after ZAPD extraction

Reply:
Yeah, sure. This is `gIceArrowTex`:
![ice_arrow_tex_1 ia8](https://user-images.githubusercontent.com/12245827/149441184-5f1a28cd-dbb4-4ac7-861f-9afac50d322d.png)

And this is `gIceArrowMaskTex`:
![ice_arrow_tex_2 ia8](https://user-images.githubusercontent.com/12245827/149441206-77e9e8b6-45c4-42e6-a2cb-b217dea13b03.png)